openfeature-provider/java/STICKY_RESOLVE.md

```diff
@@ -0,0 +1,91 @@
+# Sticky Resolve Documentation
```

Comment:
TODO: Verify if we still need this given that we have other Sticky documentation in this repo